01 - Getting Started.ipynb

```diff
@@ -24,13 +24,13 @@
     "### `conda` / `mamba`\n",
```

Comment:
<p>Regarding the two solutions, is there a preferred way  (performance wise or best practice) between <code>.mutate</code> and <code>.drop</code> and <code>.select</code> . Maybe add a note about it?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Good call -- using this as an opportunity to mention that the SQL we write and whatever the engine executes can be different things</p>